lib/constants/locales.js

```diff
@@ -10,17 +10,17 @@
 
 // Please keep English at the top, and sort other entries by `name`.
 export default {
-  en: { name: 'English' },
-  ca: { name: 'Catalan', nativeName: 'Català', completion: '28%' },
-  zh: { name: 'Chinese', nativeName: '中文', completion: '26%' },
-  cs: { name: 'Czech', nativeName: 'čeština', completion: '21%' },
-  nl: { name: 'Dutch', nativeName: 'Nederlands', completion: '22%' },
-  fr: { name: 'French', nativeName: 'Français', completion: '100%' },
-  de: { name: 'German', nativeName: 'Deutsch', completion: '25%' },
-  it: { name: 'Italian', nativeName: 'Italiano', completion: '44%' },
-  ja: { name: 'Japanese', nativeName: '日本語', completion: '17%' },
-  ko: { name: 'Korean', nativeName: '한국어', completion: '22%' },
-  pt: { name: 'Portuguese', nativeName: 'Português', completion: '86%' },
-  ru: { name: 'Russian', nativeName: 'Русский', completion: '29%' },
-  es: { name: 'Spanish', nativeName: 'Español', completion: '55%' },
+  en: { name: 'English', nativeName: 'English', completion: '100%', flag: '🇺🇸' },
+  ca: { name: 'Catalan', nativeName: 'Català', completion: '28%', flag: '🇦🇩' },
```

Comment:
This is Andorra, not Catalonia.

Reply:
I'll take there of that, thanks

Reply:
Andorra is actually the main country for Catalan, so using 🇦🇩 should be fine